01_prepare_host.sh

```diff
@@ -79,6 +79,12 @@ if ! command -v kustomize 2>/dev/null ; then
     rm "kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz"
 fi
 
+if ! [ -x "$(command -v clusterctl)" ]; then
+  curl -L -O "https://github.com/kubernetes-sigs/cluster-api/releases/download/${CLUSTERCTL_VERSION}/clusterctl-linux-amd64" -o clusterctl
```

Comment:
```suggestion
  curl -L -O https://github.com/kubernetes-sigs/cluster-api/releases/download/"${CLUSTERCTL_VERSION}"/clusterctl-linux-amd64 -o clusterctl
```
Maybe this will work?

Reply:
it might be slightly more readable with the quotes outside ?